README.md

```diff
@@ -5,143 +5,13 @@ There are currently five working endpoints:
 <li>"<code>/repository/files/</code>" returns the index search results along with a count of the terms available for the facets.</li>
 <li>"<code>/repository/files/summary</code>" returns a summary of the current data stored.</li>
 <li>"<code>/repository/files/export</code>" returns a manifest file with the filters provided.</li>
-<li>"<code>/repository/files/meta</code>" returns the desired order for the facets.</li>
+<li>"<code>/repository/files/order</code>" returns the desired order for the facets.</li>
 <li>"<code>/keywords</code>" returns a list of search results for some search query.</li>
 </ul>
 
 <h2>Installation Instructions</h2>
 
-Please refer to [dcc-ops](https://github.com/BD2KGenomics/dcc-ops) for installing the the dashboard-service and all its required components. This will automatically set up the dashboard service using an NGINX server.
-
-<h2>Developer Installation Instructions; Apache Install</h2>
-## Assumptions 
-<ul>
-<li>You have Elasticsearch 5.0.0 installed in your machine.</li>
-<li>You have virtualenv installed in your machine. </li>
-</ul>
-
-If you don’t have virtualenv installed, you can do so by running:
-```
-sudo easy_install virtualenv
-```
-## Setting up the Directory Structure
-The following directory structure should be set up:
-
-```
-|
-|__dcc-dashboard-service
-|
-|__dcc-metadata-indexer
-|
-|__redwood-client
-	|
-	|__ucsc-storage-client
-		|
-		|__...etc
-```
-To  get the dcc-dashboard-service, do:
-```
-git clone https://github.com/BD2KGenomics/dcc-dashboard-service.git
-```
-It is assumed that you’ve setup the other folders (the dcc-metadata-indexer, and the redwood-client). If you haven’t, see https://github.com/BD2KGenomics/dcc-metadata-indexer/tree/release/1.0.0 to set up the dcc-metadata-indexer. The link for the redwood-client is also in the same page. You can set the redwood-client directory and download the client by running the following commands:
-```
-#Make the redwood-client directory and cd into the directory
-mkdir redwood-client
-cd redwood-client
-#And then just download the client
-wget https://s3-us-west-2.amazonaws.com/beni-dcc-storage-dev/20161216_ucsc-storage-client.tar.gz && tar zxf 20161216_ucsc-storage-client.tar.gz && rm -f 20161216_ucsc-storage-client.tar.gz
-```
-To set up the dashboard, follow the instructions https://github.com/BD2KGenomics/dcc-dashboard
-
-## Installing Apache
```

Comment:
You had mentioned to me that we would not need Apache anymore, now that we use NGINX. Is it still needed for some other functionality?

Reply:
Let me update the README